examples/oracle.yml

```diff
@@ -20,7 +20,8 @@ resource_types:
       # SQL query to fetch user-related data
       query: |
         SELECT
-          USERNAME, USER_ID, ACCOUNT_STATUS, CREATED, LAST_LOGIN
+          USERNAME, USER_ID, ACCOUNT_STATUS, CREATED, LAST_LOGIN, 
+          EMPLOYEE_ID  -- Add employee ID field
```

Comment:
This doesn't exist on the actual Oracle schema, I don't think we should have it here?

Reply:
https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/DBA_USERS.html